src/models/data/loginData.ts

```diff
@@ -8,6 +8,7 @@ export class LoginData {
     password: string;
     passwordRevisionDate: string;
     totp: string;
+    autofillOnPageLoad: boolean;
```

Comment:
Does it matter that your server model is `nullable` here?

Reply:
It's intentional - `true` is autofill, `false` is don't autofill, `null` is treated as "not configured (therefore use default settings)". I originally had an enum for this, but I thought that was overkill when a `bool` would do the job.

Reply:
Yeah, for whatever reason I was thinking you had to define `boolean | null` but you don't, so yeah, it's my dumb 🤦🏻